### PR TITLE
Fix docs for `draw_indexed_indirect`

### PR DESCRIPTION
**Pull request by icefoxen**

### CODE REVIEW

File: command/src/buffer/encoder.rs [hunk 1]
```diff
@@ -336,12 +336,12 @@ where
...
-    /// Similar to [`draw`] except takes vertices and indices from `buffer` at specified `offset`.
-    /// `buffer` must contain `draw_count` of [`DrawCommand`] starting from `offset` with `stride` bytes between each.
+    /// Draw indirect with indices.
+    /// Similar to [`draw_indexed`] except takes vertices, indices and instance data from `buffer` at specified `offset`.
+    /// `buffer` must contain `draw_count` of [`DrawIndexedCommand`] starting from `offset` with `stride` bytes between each.
     ///
-    /// [`draw`]: trait.RenderPassInlineEncoder.html#tymethod.draw
-    /// [`DrawCommand`]: struct.DrawCommand.html
+    /// [`draw`]: trait.RenderPassInlineEncoder.html#tymethod.draw_indexed
+    /// [`DrawCommand`]: struct.DrawIndexedCommand.html
```
Comment (omni-viral) [inline command/src/buffer/encoder.rs]: "[`DrawIndexedCommand`]: struct.DrawIndexedCommand.html"

Comment (icefoxen) [reply command/src/buffer/encoder.rs]: Heck.

**Comment by omni-viral** [comment]
You're fast :)

LGTM!

bors r+

**Comment by omni-viral** [comment]
bors retry

**Pull request merged**